stages/org.osbuild.bootc.install-to-filesystem

```diff
@@ -46,6 +46,23 @@ def main(options, inputs, paths):
         stateroot = options.get("stateroot")
         if stateroot:
             pargs.extend(["--stateroot", stateroot])
+        # Passing the flag with an empty value is intentional:
+        # it triggers a supported behaviour where mountspec
+        # kernel arguments are ommited.
+        # See https://github.com/bootc-dev/bootc/pull/1451
+        if "root-mount-spec" in options:
+            root_spec = options["root-mount-spec"]
+            if root_spec == "":
+                pargs.extend(["--root-mount-spec="])
+            else:
+                pargs.extend(["--root-mount-spec", root_spec])
```

Comment:
Nitpick: could this be simplified to:
```python
pargs.extend([f"--root-mount-spec={options["root-mount-spec"]}")
```
?

And likewise below.

Reply:
Yes, the whole block would be equivalent.  The comment is good information though, so please keep that.